agent/firewall.go

```diff
@@ -113,23 +117,26 @@ func (fw *Firewall) Init(netif NetIf) error {
 	// Allow ICMP, DHCP and SSH between host and instances.
 	hostAddr := fw.Agent.networkConfig.romanaGW
 	inputRules := []string{
-		fmt.Sprintf("-d %s/32 -p icmp -m icmp --icmp-type 0 -m state --state RELATED,ESTABLISHED", hostAddr),
-		fmt.Sprintf("-d %s/32 -p tcp -m tcp --sport 22", hostAddr),
 		"-d 255.255.255.255/32 -p udp -m udp --sport 68 --dport 67",
```

Comment:
We have a special function for the creation of k8s endpoints, and another for the creation of OS endpoints. Maybe those functions should be able to pass rules to the firewall function. That way, we don't have to have unnecessary rules for k8s and all environment specific stuff is contained in those two environment specific endpoint creation functions.